Sources/Kafka/Configuration/KafkaConfiguration+Security.swift

```diff
@@ -309,7 +309,7 @@ extension KafkaConfiguration {
             }
         }
 
-        public struct OAuthBearerMethod: Sendable, Hashable {
+        struct OAuthBearerMethod: Sendable, Hashable {
```

Comment:
Won't it be (formally) a breaking change if anyone uses that one for auth?

Reply:
Yes it would but we haven't released a major yet so removing APIs is still possible. I would love to get the package into a state where we don't require system dependencies at all.